app/models/customer.rb

```diff
@@ -3,4 +3,13 @@ class Customer < ApplicationRecord
   validates :last_name, presence: true
   
   has_many :invoices, dependent: :destroy
+  has_many :transactions, through: :invoices
+
+  def self.top_customers
```

Comment:
Should we potentially change this method name to 'top_five_customers'? Or does that feel unnecessary?

Reply:
This is done, I changed it everywhere